content/en/blog/_posts/2022-08-04-cgroupv2-ga.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: blog
+title: "Kubernetes 1.25: cgroup v2 graduates to GA"
+date: 2022-08-31
+slug: cgroupv2-ga-1-25
+---
+
+**Authors:**: David Porter (Google), Mrunal Patel (RedHat)
+
+Kubernetes 1.25 brings cgroup v2 to GA (general availability), letting the
+kubelet use the latest resource management capabilities.
+
+## What are cgroups?
+
+Effective [resource management](/docs/concepts/configuration/manage-resources-containers/) is a
+critical aspect of Kubernetes. This involves managing the finite resources in
+your nodes, such as CPU, memory, and storage.
+
+*cgroups* are a Linux kernel capability that establish resource management
+functionality like limiting CPU usage or setting memory limits for running
+processes.
+
+When you use the resource management capabilities in Kubernetes, such as configuring
+[requests and limits for Pods and containers](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits),
+Kubernetes uses cgroups to enforce your resource requests and limits.
+
+The Linux kernel offers two versions of cgroups: cgroup v1 and cgroup v2.
+
+## What is cgroup v2?
+
+cgroup v2 is the latest version of the Linux cgroup API. cgroup v2 provides a
+unified control system with enhanced resource management capabilities.
+
+cgroup v2 has been development in the Linux Kernel since 2016 and in recent
+years has matured across the container ecosystem. With Kubernetes 1.25, cgroup
+v2 support is graduating to GA.
+
+Many new Linux distributions have switched over to cgroup v2 by default so it's
+important that Kubernetes continues to work well on these new updated distros.
+
+cgroup v2 offers several improvements over cgroup v1, such as the following:
+
+* Single unified hierarchy design in API
+* Safer sub-tree delegation to containers
+* Newer features like Pressure Stall Information
+* Enhanced resource allocation management and isolation across multiple resources
+    * Unified accounting for different types of memory allocations (network and kernel memory, etc)
+    * Accounting for non-immediate resource changes such as page cache write backs
+
+Some Kubernetes features exclusively use cgroup v2 for enhanced resource
+management and isolation. For example,
+the [MemoryQoS feature](https://kubernetes.io/blog/2021/11/26/qos-memory-resources/) improves
+memory QoS and relies on cgroup v2 functionality to enable it. New resource
```

Comment:
nit: would be good to reword. Currently this is a bit tautological and doesn't inform readers who haven't encountered the need for QoS around containers and RAM.

What word(s) should go after memory (eg: performance, bus read latency, allocation reliability)? I think the effect is to reduce the impact of one container / Pod on another container / Pod's allocations, but it's been a while since I looked at that feature. A good summary would be nice.

Reply:
I updated a bit, I think "utilization" might be one way we can describe it